tidb-binlog/binlog-control.md

```diff
@@ -0,0 +1,138 @@
+---
+title: binlogctl
+summary: Learns how to use `binlogctl`.
+category: reference
+---
+
+# binlogctl
+
+[Binlog Control](https://github.com/pingcap/tidb-binlog/tree/master/binlogctl) (`binlogctl` for short) is a command line tool for TiDB Binlog. You can use `binlogctl` to manage TiDB Binlog clusters.
+
+`binlogctl` can be used to:
+
+* check the state of Pump or Drainer
+* pause or close Pump or Drainer
+* handle the abnormal state of Pump or Drainer
+
+The following are its usage scenarios:
+
+* An error occurs during data replication or you need to check the running state of Pump or Drainer.
+* You need to pause or close Pump or Drainer when maintaining the cluster.
+* Pump or Drainer process is exited abnormally, while the node state is not updated or is unexpected. This affects the data replication task.
+
+## Download `binlogctl`
+
+{{< copyable "shell-regular" >}}
+
+```bash
+wget https://download.pingcap.org/tidb-{version}-linux-amd64.tar.gz &&
+wget https://download.pingcap.org/tidb-{version}-linux-amd64.sha256
+```
+
+To check the file integrity, execute the following command. If the result is OK, the file is correct.
+
+{{< copyable "shell-regular" >}}
+
+```bash
+sha256sum -c tidb-{version}-linux-amd64.sha256
+```
+
+For TiDB v2.1.0 GA or later versions, `binlogctl` is integrated into the TiDB download package. For earlier versions, you need to download binlogctl separately:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+wget https://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz &&
+wget https://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.sha256
+```
+
+To check the file integrity, execute the following command. If the result is OK, the file is correct.
+
+{{< copyable "shell-regular" >}}
+
+```bash
+sha256sum -c tidb-enterprise-tools-latest-linux-amd64.sha256
+```
+
+## Descriptions
+
+Command line parameters:
+
+```
+Usage of binlogctl:
+-V
+    Outputs the binlogctl version information
+-cmd string
+    The command mode, including "generate_meta" (deprecated), "pumps", "drainers", "update-pump" ,"update-drainer", "pause-pump", "pause-drainer", "offline-pump", and "offline-drainer"
+-data-dir string
+    The file path where the checkpoint file of Drainer is stored ("binlog_position" by default) (deprecated)
+-node-id string
+    The ID of Pump or Drainer
+-pd-urls string
+    The address of PD. If multiple addresses exist, use "," to separate each ("http://127.0.0.1:2379" by default)
+-ssl-ca string
+    The file path of SSL CAs
+-ssl-cert string
+    The file path of the X509 certificate file in the PEM format
```

Comment:
Is this line's indentation correct? It is not the same as that in Chinese.
+@WangXiangUSTC